api/src/main/resources/marquez/db/migration/V46__dataset_symlinks.sql

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+CREATE TABLE dataset_symlinks (
+  dataset_uuid      UUID,
+  name              VARCHAR(255) NOT NULL,
+  namespace_uuid    UUID REFERENCES namespaces(uuid),
+  type              VARCHAR(64),
```

Comment:
Do we need `type` defined in `dataset_symlinks`? Or, would it be helpful to know the type of symlink?